pom.xml

```diff
@@ -412,9 +412,9 @@
 				<version>5.0.11</version>
 			</dependency>
 			<dependency>
-				<groupId>postgresql</groupId>
+				<groupId>org.postgresql</groupId>
 				<artifactId>postgresql</artifactId>
-				<version>9.0-801.jdbc4</version>
+				<version>42.2.5</version>
```

Comment:
Is there any reason why you did not use the latest version?

Reply:
I went through the list of [versions](https://mvnrepository.com/artifact/org.postgresql/postgresql) of type 42.x.x and the one which was used by most people, I used here. If required I can try with the latest version and report the results.

Reply:
What were the results?

Reply:
Initialization is successful. All is working fine. I used the latest 42.2.13 . Which driver I should keep 42.2.13 or 42.2.5 ?